Resources/Private/Language/locallang.xlf

```diff
@@ -425,6 +425,12 @@
 		<trans-unit id="manager.confirmBatchReleaseUnvalidated.message" xml:space="preserve">
 			<source>The selected documents will be released but not validated.</source>
 		</trans-unit>
+		<trans-unit id="manager.confirmBatchBookmark.messageHeader">
+			<source>Do you really want to bookmark the selected documents?</source>
```

Comment:
Translation <target> missing?

Reply:
The translations are in de.locallang.xlf but this is just locallang.xlf.